### PR TITLE
Improving cross-platform support

### LINKED ISSUE #53: go-fmt doesn't work on Windows
When trying to run the `go-fmt` hook on Windows, I get:
> tee: /dev/fd/5: No such file or directory

I use a local hook to get round it, but it would be nice to be able to use this repo for all the pre-commit hooks I need

### PR DESCRIPTION
**Pull request by lightglitch**
I'm opening this because #60 is without response for a long time. Also fixes #53.

### CODE REVIEW

File: run-go-fmt.sh [hunk 1]
```diff
@@ -10,5 +10,6 @@ if ! command -v gofmt &> /dev/null ; then
 fi
 
 exec 5>&1
```
Comment (dnephin) [inline run-go-fmt.sh:12]: Looks like we're not using /dev/fd/5 anymore, so this line can be removed from both scripts.

**Review by dnephin — COMMENTED** [review]
Thanks! Just one small thing to cleanup

File: run-go-fmt.sh (hunk 1 above)
Comment (lightglitch) [reply run-go-fmt.sh]: done

**Review by dnephin — APPROVED** [review]
Thanks!

**Pull request merged**